modules/social_features/social_event/modules/social_event_managers/src/Form/SocialEventManagersAddEnrolleeForm.php

```diff
@@ -292,6 +309,32 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $form['#cache']['contexts'][] = 'user';
 
+    // We should prevent add enrollments directly if social_event_max_enroll is
+    // enabled and there are no left spots.
+    if ($this->moduleHandler->moduleExists('social_event_max_enroll')) {
+      $event_max_enroll_service = $this->eventMaxEnrollService;
+
+      /** @var \Drupal\node\NodeInterface $node */
+      $node = $this->entityTypeManager->getStorage('node')->load($nid);
+
+      if (
+        $node instanceof NodeInterface &&
+        $event_max_enroll_service->isEnabled($node)
+      ) {
+        // Count how many spots are left.
+        $left = $event_max_enroll_service->getEnrollmentsLeft($node);
+
+        // If there are no spots left, disable button and add the button title
+        // with appropriate notice.
+        if ($left === 0) {
```

Comment:
Because you are not using `$left` in any other place, you could simply write it like this:

`if ($event_max_enroll_service->getEnrollmentsLeft($node) === 0)` this is pretty clear